content/hi/auto_scaling.md

```diff
@@ -0,0 +1,13 @@
+---
+title: ऑटो स्केलिंग
+status: Completed
+category: गुण
+---
+
+ऑटोस्केलिंग एक सिस्टम की स्वचालित रूप से [स्केल](/scalability/) होने की क्षमता है, आमतौर पर, कंप्यूटिंग संसाधनों के संदर्भ में। एक ऑटोस्केलिंग सिस्टम के साथ, जरूरत पड़ने पर संसाधन स्वचालित रूप से जुड़ जाते हैं और उपयोगकर्ता की उतार-चढ़ाव वाली मांगों को पूरा करने के लिए स्केल कर सकते हैं। ऑटोस्केलिंग प्रक्रिया भिन्न होती है और विभिन्न मेट्रिक्स, जैसे मेमोरी या प्रक्रिया समय के आधार पर स्केल करने योग्य होती है। प्रबंधित क्लाउड सेवाएं आमतौर पर ऑटोस्केलिंग कार्यक्षमता से जुड़ी होती हैं क्योंकि अधिकांश ऑन-प्रिमाइसेस परिनियोजन की तुलना में अधिक विकल्प और कार्यान्वयन उपलब्ध होते हैं।
+
+पहले, आधारिक संरचना और एप्लीकेशन्स को प्रणाली के चरम पर उपयोग करने के विचार से तैयार किया गया था। इस वास्तुकला का मतलब था कि अधिक संसाधनों का कम उपयोग और  बदल्ते उपभोक्ता मांग को सम्पूर्ण करने में अयोग्यता । जिसका मतलब व्यवसाय की उच्च लागत और अधिक मांग के कारण आउटेज से व्यवसाय खो जाना था।
+
+क्लाउड, [वर्चुअलाइज़िंग](/virtualization/), और [कंटेनराइज़िंग](/containerization/) एप्लिकेशन और उनकी निर्भरता का लाभ उठाकर, संगठन उपयोगकर्ता की मांगों के अनुसार बड़े पैमाने पर एप्लिकेशन बना सकते हैं। वे एप्लिकेशन की मांग की निगरानी कर सकते हैं और इष्टतम उपयोगकर्ता अनुभव प्रदान करते हुए स्वचालित रूप से उन्हें स्केल कर सकते हैं। हर शुक्रवार शाम को नेटफ्लिक्स के दर्शकों की संख्या में वृद्धि को लें। ऑटोस्केलिंग आउट का अर्थ है गतिशील रूप से अधिक संसाधन जोड़ना: उदाहरण के लिए, अधिक वीडियो स्ट्रीमिंग की अनुमति देने वाले सर्वरों की संख्या में वृद्धि और खपत सामान्य होने के बाद वापस स्केलिंग।
```

Comment:
One thing to keep in mind is that the [https://glossary.cncf.io/auto_scaling/](https://glossary.cncf.io/auto_scaling/) page does not have section like `what it is`, `problem it addresses` , etc. 

Will be opening a PR for the english page.

Reply:
Its not a bug, Terms in "property" category don't require those 3 headings. Refer style guide for more info on this.